### DISCUSSION TITLE
Bug with virtual attribute in listing shows internal IDs

### DISCUSSION THREAD

**Discussion by zssvaidar** [atrocore/atrocore#780] [Q&A] [answered]
![image](https://github.com/atrocore/atrocore/assets/30953478/f1bc2083-e056-4c0a-831c-92915a9bf70d)
For externsibleEnum as externsibleEnumOptions this field in listings when enabled shows ids instead of label from extensibleEnumOption

**Comment by rratsun** [comment, marked as answer]
Fixed as PIM 1.9.14
